tamr_unify_client/client.py

```diff
@@ -145,6 +145,22 @@ def datasets(self):
         """
         return self._datasets
 
+    def create_project(unify, project_config):
+        """
+        Create a Project in Unify
+
+        :param: Project configuration should be formatted as specified in the `Public Docs for Creating a Project <https://docs.tamr.com/reference#create-a-project>`_.
+        :type: dict
+        :returns: The created Project
+        :rtype: :class:`~tamr_unify_client.models.project.resource.Project`
+        """
+        from tamr_unify_client.models.project.resource import Project
+
+        data = (
+            unify.post(unify.projects.api_path, json=project_config).successful().json()
+        )
+        return Project(unify, data)
```

Comment:
Usually we use the `from_json` methods to instantiate models. Could you use `from_json` unless there's a specific reason to instantiate this directly using the `Project` constructor?